akka-testkit-typed/src/main/scala/akka/testkit/typed/javadsl/ActorTestKit.scala

```diff
@@ -98,7 +99,7 @@ object ActorTestKit {
  *
  * For synchronous testing of a `Behavior` see [[BehaviorTestKit]]
  */
-final class ActorTestKit protected (delegate: akka.testkit.typed.scaladsl.ActorTestKit) {
+final class ActorTestKit protected (delegate: akka.testkit.typed.scaladsl.ActorTestKit) extends Within {
```

Comment:
`ActorTestKit` doesn't have any functionality that is using the `remaining` so `Within` should not be needed here. That is one of the intentional differences between untyped and typed testskits. All message asserts and such are done with explicit probes, and the ActorTestKit is not a probe itself.